### PR TITLE
Pm 2022 01 styling issues

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Review by fuzzylogic2000 — APPROVED** [review]
Wow! Closing so many issues! :tada:

File: adhocracy-plus/assets/scss/components/_poll.scss [hunk 1]
```diff
@@ -157,6 +157,7 @@ $checkbox-size: 20px;
 
     i {
         color: $brand-tertiary;
+        margin-left: -1.25 * $spacer;
```
Comment (khamui) [inline adhocracy-plus/assets/scss/components/_poll.scss:160]: this gets a little bit cut off to the left on screens below 768px.
Oops, it got just merged. Just leaving this mention here then. @phillimorland

Comment (phillimorland) [reply adhocracy-plus/assets/scss/components/_poll.scss:160]: ah cheers!

Comment (fuzzylogic2000) [reply adhocracy-plus/assets/scss/components/_poll.scss:160]: Oh, sorry!

**Pull request merged**